doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -233,3 +233,24 @@ The leader network data.
 * Packed-Encoding: `D`
 
 The stable leader network data.
+
+### PROP 5391: PROP_THREAD_JOINERS {#prop-thread-joiners}
+
+* Type: Read-Write
+* Packed-Encoding: `A(T(DLE))`
+
+Data per item is:
+
+* `D`: PSKd
```

Comment:
@jwhui and I looked at the Thread spec about the PSKd and found it to be rather underspecified.

We'd prefer this be the base-32 encoded string be used here instead of the "binary data" representation (which is underspecified). That way we can clarify with the Thread Group and correct the problem in the NCP. It also makes the API more straightforward.

That would mean the packed-encoding would be `A(t(ULE))`.